README.rst

```diff
@@ -20,18 +20,24 @@
 Hawkmoth - Sphinx Autodoc for C
 ===============================
 
-Hawkmoth is a minimalistic Sphinx_ `C Domain`_ autodoc directive extension to
-incorporate formatted C source code comments written in reStructuredText_ into
-Sphinx based documentation. It uses Clang Python Bindings for parsing, and
-generates C Domain directives for C API documentation, and more. In short,
-Hawkmoth is Sphinx Autodoc for C.
+Hawkmoth is a minimalistic Sphinx_ `C/C++ Domain`_ autodoc directive extension
+to incorporate formatted C/C++ source code comments written in reStructuredText_
+into Sphinx based documentation. It uses Clang Python Bindings for parsing, and
+generates C/C++ Domain directives for C API documentation, and more. In short,
+Hawkmoth is Sphinx Autodoc for C/C++.
```

Comment:
I'm still frowning on the C/C++. Two languages, two domains... one wouldn't say "C/Python". :p

Reply:
I didn't invent the C/C++ monogram xD

Though, sadly, C++ is only half a new language, so I actually understand the '/'. No prejudice to what you mentioned elsewhere though! I'll use 'and' and 'or' instead.

Reply:
Obviously C/C++ is widely used, it's just a kind of pet peeve of mine, and personally I avoid it when I can.

What to reply when someone asks, "do you know C/C++?"... "Very well/not so much."

Or, "do you like C/C++"... "Yes/no." :)

Anyway, I'll live with some C/C++ mentions to avoid repetition as long as it's not the main thing. Fair?

Reply:
Ahah, no I get that and to a large extent I agree. I'd say that this particular line might be one of the few places where it's less questionable, another one would be when talking about a C/C++ compiler and so on. I.e. it really means either in the most interchangeable sense. But I agree with changing them all.

Well, I wonder if it may be helpful to leave a few for the search engines xD